docs/getting-started/user-guides/paymetheus.md

```diff
@@ -0,0 +1,60 @@
+# **Paymetheus** #
+
+Paymetheus is a simple to use graphical wallet for Decred. With it, you can send and receive DCR, 
+purchase tickets for [PoS voting](/mining/proof-of-stake.md), get a history of all your transactions and more.
+About the only thing Paymetheus doesn't do is PoS voting.
+
+---
+
+## **Download and Install** ##
+The first thing you'll need to do is get a copy of Paymetheus. You will find the downloads [here](https://github.com/decred/decred-binaries/releases). 
+Download the file for your operating system and install. Once you're done, run the 'Decred' program.
+You'll notice a program called "Paymetheus Standalone". This does not include the 'dcrd' program 
+required to connect to the Decred network, so don't use this one.
```

Comment:
saying "don't use this" is a bit harsh :)

I'd say that both serve different purposes and then, starting with the launcher version that does start dcrd, walk through how both can be used.